usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -0,0 +1,288 @@
+import itertools
+import numpy as np
+from django import db
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from pyspark.sql import SparkSession
+
+from usaspending_api.common.helpers.spark_helpers import (
+    configure_spark_session,
+    get_active_spark_session,
+    get_jdbc_connection_properties,
+    get_jdbc_url,
+    get_jvm_logger,
+)
+from usaspending_api.etl.management.commands.create_delta_table import TABLE_SPEC
```

Comment:
I'm thinking this should `TABLE_SPEC` options from `load_query_to_delta` instead of `create_delta_table`. `create_delta_table` doesn't have the new, necessary options like `swap_table` and `swap_schema`

Reply:
In that case, I'm more tempted to add `swap_table` to those and set them to `None`s for consistency across all of them.

Reply:
Just checked, looks like you did add them for those tables. I missed those earlier. I don't see why we'd want to add configuration for copies that we're never going to use though

Reply:
I was going for consistency across all the options in `TABLE_SPEC` and allowing to pull from other tables like `sam_recipient` or other tables which aren't available in the `TABLE_SPEC` in `load_query_to_delta`